src/plugins/camera/camera_impl.h

```diff
@@ -180,6 +188,11 @@ class CameraImpl : public PluginImplBase {
         Camera::Status data{};
         bool received_camera_capture_status{false};
         bool received_storage_information{false};
+        int image_count{-1};
+        int image_count_at_connection{-1};
+        int last_advertised_image_index{-1};
+        std::map<int, Camera::CaptureInfo> photo_list;
```

Comment:
Do you need an ordered map (`std::map`) or is an unordered map good enough (`std::unordered_map`)?

Reply:
I think that conceptually, the ordered map makes sense since I convert it to a vector for the user, and I want the images ordered there... not sure in terms of performance, though